app/src/main/java/org/mozilla/focus/widget/CookiesPreference.kt

```diff
@@ -31,11 +31,16 @@ class CookiesPreference(context: Context?, attrs: AttributeSet?) : ListPreferenc
 
     fun updateSummary() {
         val settings = context.settings
-        super.setSummary(settings.shouldBlockCookiesValue())
+
+        val cookieOptionIndex =
+            context.resources.getStringArray(R.array.cookies_options_entry_values)
+                .asList().indexOf(settings.shouldBlockCookiesValue())
+        this.summary =
+            context.resources.getStringArray(R.array.cookies_options_entries)[cookieOptionIndex]
```

Comment:
I'm receiving ArrayIndexOutOfBoundsException: length=5; index=-1 at this line

Reply:
Thanks for testing. I think you already had a setting saved with another locale. We have to find a solution for that, but I think it will involve resetting user changes to the default value for other locales.